### PR TITLE
Feature 162 Add integration api resume detail 

### PR DESCRIPTION
**Pull request by ela-ocana-jalafund**
[AB#165](https://dev.azure.com/funjaladev/7867a1c5-5ac7-4c7f-baff-87d740bf4fc1/_workitems/edit/165)
https://dev.azure.com/funjaladev/DevLevel3/_boards/board/t/DevLevel3%20Team/Stories/?workitem=165
# Overview
- Add navigations buttons with links routes into Resumes list. (Navigations between resumes list, create resume and resume detail)
- Add a button "New Resume" in resumes list view to create a new Resume.
- Add buttons into boxes of resumes list. Each Resume Box has a button to see Resume's detail.
- Add styles to resumes list acoording to color tools.

[AB#162](https://dev.azure.com/funjaladev/7867a1c5-5ac7-4c7f-baff-87d740bf4fc1/_workitems/edit/162)
https://dev.azure.com/funjaladev/DevLevel3/_boards/board/t/DevLevel3%20Team/Stories/?workitem=162
- Show correct information about person resume. Integrations with API should show example data of teamUp repository.
- Test Response Code: Success, Not Found, Bad Request.
- Show correct information about person resume. Integrations with API should show example data of teamUp repository.

## Outstanding Tasks
imagen

### CODE REVIEW

**Review by rodrigo-baldiviezo-jalafund — DISMISSED** [review]
LGTM

**Review by niver-copa-jalafund — DISMISSED** [review]

File: src/components/resumes/ResumesAPI.js [hunk 1]
```diff
@@ -6,36 +6,62 @@ const API_VERSION = "v1";
...
     );
-
-    return resumes;
+    if (result.status === 200) {
+      return { ok: true };
+    } else {
+      result.ok = false;
+      result.statusText = "Data not saved";
+    }
+    console.log(result.statusText);
```
Comment (jorge-lema-jalafund) [inline src/components/resumes/ResumesAPI.js]: remove this console.log please

File: src/components/resumes/ResumesAPI.js [hunk 2]
```diff
@@ -6,36 +6,62 @@ const API_VERSION = "v1";
 
 export async function postResume(resume) {
   try {
-    const resumes = await axios.post(
+    const result = await axios.post(
       "http://localhost:7071/api/resumes",
```
Comment (jorge-lema-jalafund) [inline src/components/resumes/ResumesAPI.js]: can you update this to follow our new pattern. 
image

File: src/components/resumes/Details.jsx [hunk 3]
```diff
@@ -34,100 +51,112 @@ function Details({ resumeId }) {
...
+          <div>
+            <TextField
+              InputLabelProps={{
+                shrink: true,
+              }}
+              defaultValue={data.contact.email}
+              fullWidth
+              label="Email"
+              variant="standard"
+            />
```
Comment (jorge-lema-jalafund) [inline src/components/resumes/Details.jsx]: This field should be readOnly too

File: src/components/resumes/Details.jsx [hunk 4]
```diff
@@ -34,100 +51,112 @@ function Details({ resumeId }) {
...
+      className={classes.content}
+      style={{
+        bottom: 130,
+        left: "30%",
+        position: "absolute",
+        top: 130,
+      }}
+    >
+      <Paper className={classes.paper}>
+        <form autoComplete="off" className={classes.root} noValidate>
```
Comment (jorge-lema-jalafund) [inline src/components/resumes/Details.jsx:88]: Could you improve the UI/UX a little? maybe Applying some styles to the fields when they are readonly or on edit mode? 
You could disable them or apply colors...or maybe change the variant of the component. It's up to you to decide :)

Comment (jorge-lema-jalafund) [reply src/components/resumes/Details.jsx:88]: Please use Formik to validate and handle the form's data.

**Review by jorge-lema-jalafund — CHANGES_REQUESTED** [review]

**Review by niver-copa-jalafund — APPROVED** [review]
Looks fine

**Review by rodrigo-baldiviezo-jalafund — APPROVED** [review]
LGTM

**Review by jorge-lema-jalafund — APPROVED** [review]
approved

**Pull request merged**